geomesa-hbase/geomesa-hbase-tools/src/main/resources/org/locationtech/geomesa/hbase/tools/ingest-libjars.list

```diff
@@ -22,8 +25,11 @@ mango
 metrics
 minlog
 netty
+paranamer
 parboiled
+Saxon-HE
```

Comment:
What happens if the user has *not* installed the Saxon-HE jar?

Reply:
won't get pulled. just skipped.

Reply:
https://github.com/locationtech/geomesa/blob/master/geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/classpath/ClassPathUtils.scala#L33